usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -667,6 +677,8 @@ This endpoint returns a list of data that is associated with the award profile p
     `null` if the contract has no parent
 + `naics_hierarchy` (required, NAICSHierarchy, fixed-type)
 + `psc_hierarchy` (required, PSCHierarchy, fixed-type)
++ `national_interest_action` (required, string)
```

Comment:
Should these also be `nullable`?

Reply:
Yes, unsure how I missed that

Reply:
Ha, they were defined in the wrong spot too. Corrected both issues

Reply:
TY!